src/components/App/App.tsx

```diff
@@ -12,32 +12,76 @@ import React, { FC } from "react";
 import "./App.css";
 
 const App: FC = () => {
-    const { isDarkMode, updateName, name } = useSignatureContext();
+    const { isDarkMode, updateState, name, email, company, title, linkedin, phone } = useSignatureContext();
     const { profile } = useAuthenticationContext();
 
     return (
-        <ProviderV3 theme={isDarkMode ? darkTheme : lightTheme} colorScheme={"light"} height="100%">
-            <ToastContainer />
-            <Layout>
-                <Flex marginX="size-300" alignItems="start" direction="column" justifyContent="center" gap={"size-200"}>
-                    {profile ? <h3>Welcome {profile?.given_name}</h3> : <></>}
```

Comment:
@ramirezStefano

Reply:
yes! the removed {profile ? <h3>Welcome {profile?.given_name}</h3> : <></>} line should be replaced as it is use for the Google sign in implementation and use 'profile' within the flex component wrapper and before the first UsernameInput component as to keep the UI esthetics 👾